modules/apps/headless-apio/forms/forms-apio-impl/src/main/java/com/liferay/forms/apio/internal/architect/provider/ServiceContextWrapperProvider.java

```diff
@@ -34,18 +34,18 @@
  * @author Paulo Cruz
  */
 @Component(immediate = true)
-public class FormInstanceRecordServiceContextProvider
-	implements Provider<FormInstanceRecordServiceContext> {
+public class ServiceContextWrapperProvider
```

Comment:
I think it's right, because is related to forms, but most of the providers are on the portal module. (just check)